src/cpp/client/ProxyClient.cpp

```diff
@@ -36,12 +36,14 @@ namespace uxr {
 
 ProxyClient::ProxyClient(
         const dds::xrce::CLIENT_Representation& representation,
-        Middleware::Kind middleware_kind)
+        Middleware::Kind middleware_kind,
+        std::unordered_map<std::string, std::string> properties)
```

Comment:
Same here, I would change to `std::unordered_map<std::string, std::string>*`

Reply:
Done